ospo-mindmap/Content/ospomindmap.md

```diff
@@ -109,24 +189,87 @@ incubation and post launch
   - Vet, organize and operate OS projects
   - Train and coach OS Leaders
 
-### OS Leadership
+#### Optimize Open Outbound Source Contributions
+
+
+### 🤝 Collaborate with Open Source Organizations
+
+##### 📖 Open Standards
+
+- OpenChain
+- SPDX
+
+##### 🏠 Foundations
+
+- Apache Foundation
+- Free Software Foundation Europe
+  - Legal Network
+- Eclipse Foundation
+- Linux Foundation
+  - FINOS
+  - OpenSSF
+  - OpenChain
+  - SPDX
+  - TODO Group
+- Open Source Initiative
+- OW2
+  - Good Governance Initiative
 
-#### Provides open source consultancy and become a strategic partner
 
-- Evaluate the behavior
-and performance of the project
-- Advise on which open source technologies
-to adopt or remove
-- Provide insights into OSS trends
+### ⏱️ Track Performance Metrics
+
+- Ability to assess health of open source projects
+- Measure community engagement
+- Development of open source leaders
+
```

Comment:
I have a couple of slides on a slide deck talking about performance metrics for individuals contributing to upstream.
What about adding:
- Measure impact upstream

Reply:
Changed  Measure community engagement  ->  Measure community engagement / impact upstream